index.js

```diff
@@ -1,44 +1,40 @@
 module.exports = {
-    env: {
-        browser: true,
-        node: true,
-        es2024: true,
-        mocha: true
+    name: "tao:js",
+    files: ["**/*.{cjs,mjs,js,svelte}"],
+    languageOptions: {
+        globals: {
+            browser: true,
+            node: true,
+            es2024: true,
+            mocha: true,
+            Cypress: true,
+            cy: true,
+        },
     },
-    globals: {
-        Cypress: true,
-        cy: true
-    },
-    parser: '@babel/eslint-parser',
-    parserOptions: {
-        sourceType: 'module',
-        requireConfigFile: false
-    },
-    extends: 'eslint:recommended',
```

Comment:
I would keep the eslint recommended rules, as we do [here](https://github.com/oat-sa/eslint-config-tao/blob/master/flat/svelte.mjs#L15) for the frontend (svelte) config. In this PR you are removing those rules